Makefile

```diff
@@ -282,10 +282,10 @@ SHLIB_FLAVOUR = cygdll
 lib-shared: cyghts-$(LIBHTS_SOVERSION).dll
 else ifeq "$(findstring MSYS,$(PLATFORM))" "MSYS"
 SHLIB_FLAVOUR = dll
-lib-shared: hts-$(LIBHTS_SOVERSION).dll
+lib-shared: hts-$(LIBHTS_SOVERSION).lib
```

Comment:
They are all end products, but I'm not sure what benefit there is to explicitly specifying everything in the dependency chain rather than the final item.

If for some inexplicable reason you decide you want hts-3.dll only and not the .def or .lib, you can still do `make hts-3.dll` and it'll build it.  If you want to just let it build a shared libary then `make lib-shared` will do the right thing.

Reply:
Documentation essentially — it's a bit confusing if `lib-shared` lists a _.lib_ but not _.dll_.